### DISCUSSION TITLE
Nordigen json file date range

### DISCUSSION THREAD

**Discussion by joseluislucio** [firefly-iii/firefly-iii#6393] [Firefly III]
Hello.

I have created a json file to import transactions from Nordigen. Date range was 1 day. But today I wanted to import las 7 days, so I edited my json file and changed     

"nordigen_max_days": "1",

to

"nordigen_max_days": "3",

This is not working. Looks like it is still using 1 day range.

Any advise?

**Comment by JC5** [comment]
It's possible this value is retained by nordigen itself. Create a new configuration file. My suggestion would be to leave the value at the default and use the data importer settings to limit how many transactions you import.
